SwiftDungeon/GameScene.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  GameScene.swift
+//  SwiftDungeon
+//
+//  Created by Toro Juan D. on 2/28/18.
+//  Copyright © 2018 Toro Juan D. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene {
+    
+    var gameManager: GameManager?
+    
+    var stickActive:Bool = false
+    
+    override func didMove(to view: SKView) {
+        super.didMove(to: view)
+        backgroundColor = #colorLiteral(red: 0.1631777585, green: 0.1484030187, blue: 0.2081771195, alpha: 1)
```

Comment:
Hardcoded values like this in the code are generally discouraged, do to the difficulty of updating place them in a constants file and give it a name you can reference. If you reuse the color later it will be much easier to lookup.